docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+    let chainBTransaction = {"to":"0x2dcA1a80c89c81C37efa7b401e2d20c1ED99C72F","gasLimit":"60000","from":"0x291B0E3aA139b2bC9Ebd92168575b5c6bAD5236C","data":"0xa9059cbb00000000000000000000000048e6a467852fa29710aaacdb275f85db4fa420eb0000000000000000000000000000000000000000000000006f05b59d3b200000","type":2}
+    let chainAClawbackTransaction = {"to":"0xad50f302a957C165d865eD398fC3ca5A5A2cDA85","gasLimit":"60000","from":"0x48e6a467852Fa29710AaaCDB275F85db4Fa420eB","data":"0xa9059cbb00000000000000000000000048e6a467852fa29710aaacdb275f85db4fa420eb0000000000000000000000000000000000000000000000003782dace9d900000","type":2}
+    let chainBClawbackTransaction = {"to":"0x2dcA1a80c89c81C37efa7b401e2d20c1ED99C72F","gasLimit":"60000","from":"0x291B0E3aA139b2bC9Ebd92168575b5c6bAD5236C","data":"0xa9059cbb000000000000000000000000291b0e3aa139b2bc9ebd92168575b5c6bad5236c0000000000000000000000000000000000000000000000006f05b59d3b200000","type":2}
+    
+    chainATransaction.from = chainBTransaction.from = pkpAddress;
+
+    chainACondition.parameters = chainBCondition.parameters = [
+      pkpAddress,
+    ];
+
+    chainATransaction = {...chainATransaction, ...chainAGasConfig}
+    chainBTransaction = {...chainBTransaction, ...chainBGasConfig}
+    chainAClawbackTransaction = {...chainAClawbackTransaction, ...chainAGasConfig}
+    chainBClawbackTransaction = {...chainBClawbackTransaction, ...chainBGasConfig}
+```
+
+Here, we are substituting the `from` field in the transaction objects with `pkpAddress` and replacing the `parameters` the field in the conditions object with `pkpAddress`.
+
+We then, extend our transaction object with the gas configs which will be received during the execution of the action. We'll look into this later.
+
+```js
+    const chainAConditionsPass = await Lit.Actions.checkConditions({
+      conditions: [chainACondition],
+      authSig: JSON.parse(authSig),
+      chain: chainACondition.chain,
+    });
+  
+    const chainBConditionsPass = await Lit.Actions.checkConditions({
+      conditions: [chainBCondition],
+      authSig: JSON.parse(authSig),
+      chain: chainBCondition.chain,
+    });
+
+    console.log("chainAConditionsPass: ", chainAConditionsPass, "chainBConditionsPass: ", chainBConditionsPass);
+```
+
+Now our action checks if the conditions are passed on both chains, it takes `authSig` as a parameter here and then logs the result of both condition checks. We'll look into auth sigs while we learn to execute lit actions.
+
+For generating signatures inside our Lit Action, we'll use `Lit.Actions.signEcdsa`. This method allows us to use a pkp (which lets an action generate a sign) for generating signatures. Our Lit Action will now decide based on checked conditions and generate appropriately signed transactions which we can send to the blockchain later.
```

Comment:
> For generating signatures inside our Lit Action, we'll use `Lit.Actions.signEcdsa`. This method allows us to use a pkp (which lets an action generate a sign) for generating signatures. Our Lit Action will now decide based on checked conditions and generate appropriately signed transactions which we can send to the blockchain later.

Consider "To generate signatures inside of our Lit Action, we'll use the `Lit.Actions.signEcdsa` method. Our Lit Action will verify that the conditions we've specified are met before generating the appropriate signed transactions which we'll be able to send to chain later"